mssqlcli/i18n.py

```diff
@@ -0,0 +1,62 @@
+import gettext
+import os
+from importlib import import_module
+import utility
+import polib
```

Comment:
We cannot import babel defined modules in this file since babel is a dev dependency and not a runtime dependency. Let's move all babel related code into build.py